packages/solid-form/vite.config.ts

```
@@ -1,10 +1,20 @@
 import { defineConfig, mergeConfig } from 'vitest/config'
 import { tanstackViteConfig } from '@tanstack/vite-config'
+import { fileURLToPath } from 'node:url'
```

ESLint

[error] `node:url` import should occur before import of `vitest/config`

(import/order)